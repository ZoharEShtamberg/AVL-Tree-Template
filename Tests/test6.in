add_country 1 0
add_team 11 1 1
add_contestant 1 1 1 800
add_contestant 2 1 1 700
add_contestant 3 1 1 600
add_contestant 4 1 1 500
add_contestant 10 1 1 80
add_contestant 20 1 1 70
add_contestant 30 1 1 60
add_contestant 40 1 1 50
add_contestant 100 1 1 8
add_contestant 200 1 1 7
add_contestant 300 1 1 6
add_contestant 400 1 1 5
add_contestant_to_team 11 1
add_contestant_to_team 11 2
add_contestant_to_team 11 3
add_contestant_to_team 11 4
add_contestant_to_team 11 10
add_contestant_to_team 11 20
add_contestant_to_team 11 30
add_contestant_to_team 11 40
add_contestant_to_team 11 100
add_contestant_to_team 11 200
add_contestant_to_team 11 300
add_contestant_to_team 11 400
update_contestant_strength 1 -200
get_team_strength 11
update_contestant_strength 1 200
get_team_strength 11