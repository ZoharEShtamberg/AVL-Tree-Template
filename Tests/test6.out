add_country: SUCCESS
add_team: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
update_contestant_strength: SUCCESS
get_team_strength: SUCCESS, 788